boards/arm/mx8mp/verdin-mx8mp/configs/nsh/defconfig

```diff
@@ -5,23 +5,43 @@
 # You can then do "make savedefconfig" to generate a new defconfig file that includes your
 # modifications.
 #
+# CONFIG_ARCH_LEDS is not set
 # CONFIG_ARCH_RAMFUNCS is not set
 # CONFIG_STANDARD_SERIAL is not set
 CONFIG_ARCH="arm"
 CONFIG_ARCH_BOARD="verdin-mx8mp"
 CONFIG_ARCH_BOARD_VERDIN_MX8MP=y
+CONFIG_ARCH_BUTTONS=y
 CONFIG_ARCH_CHIP="mx8mp"
 CONFIG_ARCH_CHIP_MX8MP=y
 CONFIG_ARCH_INTERRUPTSTACK=2048
+CONFIG_ARCH_IRQBUTTONS=y
 CONFIG_ARCH_STACKDUMP=y
 CONFIG_ARMV7M_DCACHE=y
 CONFIG_ARMV7M_DCACHE_WRITETHROUGH=y
 CONFIG_ARMV7M_ICACHE=y
 CONFIG_ARM_MPU=y
 CONFIG_BOARD_LOOPSPERMSEC=159972
 CONFIG_BUILTIN=y
+CONFIG_CXX_EXCEPTION=y
```

Comment:
do you really need cxx stuff by default?

Reply:
For my current usage yes :)

I may add another defconfig later though.